src/event/event.php

```diff
@@ -19,10 +27,10 @@ class Event extends Resource
 
     ## Attributes:
         - id [string]: unique id returned when the event is created. ex: "5656565656565656"
-        - log [Log]: a Log object from one the subscription services (Transfer\Log, Boleto\Log, BoletoPayment\log or UtilityPayment\Log)
+        - log [Log]: a Log object from one the subscription services (PixClaim\Log, IssuingCard\Log, PixKey\log or IssuingInvoice\Log)
```

Comment:
#trocar
`- log [Log]: a Log object from one the subscription services (PixClaim\Log, IssuingCard\Log, PixKey\log or IssuingInvoice\Log)`
para
`- log [Log]: a Log object from one the subscription services ex: IssuingCard\Log, PixRequest\Log`